src/openrct2/world/Climate.cpp

```diff
@@ -181,7 +181,7 @@ extern "C"
         else if (gClimateCurrentWeatherEffect == 2)
         {
             // Create new thunder and lightning
-            uint32 randomNumber = util_rand();
+            uint32 randomNumber = scenario_rand();
```

Comment:
I need to double check this, but the thunder and lightning effects should be purely client side and aesthetic.